docs/Utils/Convert_md2pdf.py

```diff
@@ -0,0 +1,31 @@
+import os 
+import pandoc
+from markdown import markdown
+import pdfkit
+import re
+#Split all .md in one
+directory = '/home/agovorov/Desktop/RequirementsTest/docs/Guide_index/' 
```

Comment:
@antongovorov Can we change it to use path as parameter or add information about the location from where this script should be run?